conventions/documenting_code.md

```diff
@@ -113,6 +113,41 @@ class Unicorn
 end
 ```
 
+### Flagging Classes, Modules, and Methods
+
+Given a valid keyword, Crystal will automatically generate visual flags that help highlight problems, notes and/or possible issues.
+
+The supported flag keywords are:
+
+- BUG
+- DEPRECATED
+- FIXME
+- NOTE
+- OPTIMIZE
+- TODO
+
+Flag keywords must be the first word in their respective line and must be in all caps.
+
+Although a trailing colon can help with readability in the code, it will be removed in the generated doc. A trailing description of the flag is optional (but helpful).
```

Comment:
Personally I would merge these two paragraphs as:
> Flag keywords must be the first word in their respective line and must be in all caps. An optional trailing colon is preferred for readability.

then let the example do the rest of the talking.